HISTORY.rst

```diff
@@ -10,6 +10,8 @@ Release History
 
 - Python 2.6 is no longer supported.
 - Python 3.3 is no longer supported.
+- ``client.search()`` now returns a ``Search`` object that exposes a ``query()`` method to call the Search API.
```

Comment:
What's the reasoning behind this change? Do we think the `Search` object may have more functionality in the future?